deploy/olm-catalog/knative-eventing-operator/0.6.0/knative-eventing-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -84,9 +79,6 @@ spec:
                 - knative-eventing-operator
                 env:
                 - name: WATCH_NAMESPACE
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
based on the removal in PR #47, this goes here too.

Reply:
I think it's fine as it is. It's definitely worse like this, since there's no value specified for WATCH_NAMESPACE. I think that will error.